erpnext/patches/v17/add_sales_invoice_custom_fields_003.py

```
@@ -0,0 +1,51 @@
+                "description": "Link to the original sales order"
+            }
+        ]
+    }
+    
+    try:
+        create_custom_fields(custom_fields, update=True)
+        frappe.db.commit()
+        print("Custom fields added to Sales Invoice")
+        
+    except Exception as e:
+        print(f"Error: {str(e)}")
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)